js/tests.js

```diff
@@ -1,5 +1,43 @@
-QUnit.test("hello test", function(assert) {
-  assert.strictEqual(1 + 1, 2, "One plus one is two");
+QUnit.test("superHero object", function(assert) {
+  var BuffySummers = new Superhero("Buffy", 50, "Angel", true, "I\'m Buffy, the vampire slayer");
+  assert.strictEqual(BuffySummers.loveInterest, "Angel", "gets created and assigns string attribute correctly");
+  assert.strictEqual(BuffySummers.isLoveRequited, true, "assigns boolean attribute correctly");
+  //I don't think I'd have to test all of these; once I know that two data types have been successfully assigned I'm happy
 });
 
-// ADD TESTS HERE
+QUnit.test("fightScene function", function(assert) {
+  var DrHorrible = new Superhero("Dr. Horrible", 10, "Penny", false, "I\'ve got a PhD in Horribleness.");
+  var CaptainHammer = new Superhero("Captain Hammer", 11, "Penny", true, "*These* are not the hammer.");
+  assert.strictEqual(DrHorrible.strength, 10, "Strength gets assinged correctly.");
+  DrHorrible.fightSceneAgainst(CaptainHammer);
+  assert.strictEqual(DrHorrible.strength, 9, "results in strength loss after losing a fight");
+  assert.strictEqual(CaptainHammer.strength, 11, "results in no strength loss after winning a fight");
+
+});
+
+
+
+QUnit.test("loveScene function", function(assert) {
+  var BlackWidow = new Superhero("Black Widow", 1000, "Hawkeye", true, "something russian idk");
+  var LoveSceneStr = BlackWidow.LoveScene();
+  assert.strictEqual(LoveSceneStr, "Black Widow approaches Hawkeye and (after obtaining consent) kisses them, and whispers \"something russian idk.\"" , "returns love scene string correctly");
+});
+
+QUnit.test("subclass Batman numParents", function(assert) {
+  assert.strictEqual(Batman.prototype.numParents(), 0 , "Batman doesn't have any parents (they were killed)");
```

Comment:
If you add methods to the prototype, typically you'll only use them from an instance. A clearer example of this might be

```javascript
var batman = new OrphanHero();
assert.strictEqual(batman.numParents(), 0);
```

If it's a class method, you would say something like

```javascript
SuperHero.all();
```